### ISSUE TITLE
iqtree and iqtree2 should be two distinct packages

### ISSUE THREAD

**Issue by pvanheus** [bioconda/bioconda-recipes#21964]
The author of IQTREE has changed the binary name to `iqtree2` from version 2.0.4 (see this [issue](https://github.com/Cibiv/IQ-TREE/issues/131)), making iqtree vs iqtree2 similar to e.g. raxml vs raxml-ng or BLAST vs BLAST+. 

Before I go and create a `iqtree2` package I wanted to note this as an issue. To follow the author's scheme, `conda install iqtree` should install from the 1.X stream and `conda install iqtree2` should install from the 2.X stream - but bioconda already contains iqtree 2.0.3 in the iqtree package. What, then, can be done?

**Comment by bgruening** [comment]
Why do we need a new package called `iqtree2`? We can keep the `iqtree` package, or do you want to support both binaries in one environment?

**Comment by pvanheus** [comment]
Yes exactly, one might want both binaries in a single environment as they are different enough (like `raxml` and `raxml-ng` to be considered distinct tools).
